modules/react/side-panel/stories/stories.tsx

```diff
@@ -86,7 +91,7 @@ const UnorderedList = styled('ul')({
   paddingLeft: 0,
 });
 
-const AddButton = styled(IconButton)({
+const AddButton = styled(SecondaryButton)({
   margin: '0 auto',
   display: 'block',
 });
```

Comment:
Box has `display`.

https://github.com/Workday/canvas-kit/blob/master/modules/labs-react/common/lib/utils/layout.ts#L13

But if you use `const InlineBox = styled(Box) { display: 'inline-block' }` the `display` prop won't work upstream. Meaning you cant do this: `<InlineBox display="inline-flex" />`. That's true of all style props.

Reply:
ah that's probably why it gives me the error because `SecondaryButton` is a `Box` component that has `display: inline-flex` so in this case I can't add the display property? Did i understand that correctly?